### ISSUE TITLE
About train/test split for  image-wise anomaly detection in ADAM dataset

### ISSUE THREAD

**Issue by xuanli98** [WentianZhang-ML/GatingAno#1]
Thanks for your great work : Anomaly detection via gating highway connection for retinal fundus images. 
I have some questions about train/test split for  image-wise anomaly detection in ADAM dataset.
In your paper:
![image](https://github.com/WentianZhang-ML/GatingAno/assets/30352320/7a6cf1be-a269-4f69-9eec-4060fc422658)
In ADAM dataset:
![image](https://github.com/WentianZhang-ML/GatingAno/assets/30352320/7a5c2e19-0e46-42cf-a3c4-1764425e6642)
The train/test split of normal images (222/89) for image-wise anomaly detection in ADAM dataset could not be found in your code or the offical ADAM dataset. I think I must miss something. Would you like to help me figure out the question? Thank you very much.

**Comment by WentianZhang-ML** [comment]
Hi, sorry for the late reply. 

We simply set "N0001"-"N0089" images in the Non-AMD folder as the test set and the rest as the training set, so as to keep the same IDs with files in the AMD folder.

Please feel free to contact me if you have any questions.

**Comment by xuanli98** [comment]
Thank you very much

**Comment by LeeHaoRanRan** [comment]
> Thanks for your great work : Anomaly detection via gating highway connection for retinal fundus images. I have some questions about train/test split for image-wise anomaly detection in ADAM dataset. In your paper: ![image](https://private-user-images.githubusercontent.com/30352320/307255111-7a6cf1be-a269-4f69-9eec-4060fc422658.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwMzA1MTAsIm5iZiI6MTczMzAzMDIxMCwicGF0aCI6Ii8zMDM1MjMyMC8zMDcyNTUxMTEtN2E2Y2YxYmUtYTI2OS00ZjY5LTllZWMtNDA2MGZjNDIyNjU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAxVDA1MTY1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM1NDBmMGE0MTAzZDY0ZTM1MThiMjJkZWUzNjk4ZjQ1YmQ5MmE0ZmQ2ZjU4MDg0Nzc1Yjc3OTNmYTAyM2E2ODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.fDppQV7wt6zVJaFk8KGA4zbHiBMWsOaLbSnUwwxOas4) In ADAM dataset: ![image](https://private-user-images.githubusercontent.com/30352320/307255250-7a5c2e19-0e46-42cf-a3c4-1764425e6642.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwMzA1MTAsIm5iZiI6MTczMzAzMDIxMCwicGF0aCI6Ii8zMDM1MjMyMC8zMDcyNTUyNTAtN2E1YzJlMTktMGU0Ni00MmNmLWEzYzQtMTc2NDQyNWU2NjQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAxVDA1MTY1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFiOTI2NjE0ZGViM2E3NmU4YmJjNDViYjdlZTM1MzdkNmIwODE3ZWRmM2VkZmE4MjA0NjZmMTU5M2QxMmZlNWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._zAMxW0ONAxQi59iTu-5DxEunbS1hJl1SK-WgCoAbKk) The train/test split of normal images (222/89) for image-wise anomaly detection in ADAM dataset could not be found in your code or the offical ADAM dataset. I think I must miss something. Would you like to help me figure out the question? Thank you very much.

How do I get the data? Do I need to wait for the audit of the competition?

**Comment by YXtrees** [comment]
> 感谢您的出色工作：通过门控高速公路连接对视网膜眼底图像进行异常检测。我对 ADAM 数据集中用于图像异常检测的训练/测试拆分有一些问题。在您的论文中：![image](https://github.com/WentianZhang-ML/GatingAno/assets/30352320/7a6cf1be-a269-4f69-9eec-4060fc422658)在 ADAM 数据集中：![image](https://github.com/WentianZhang-ML/GatingAno/assets/30352320/7a5c2e19-0e46-42cf-a3c4-1764425e6642)在您的代码或官方 ADAM 数据集中找不到用于 ADAM 数据集中图像异常检测的正常图像的训练/测试拆分 （222/89）。我想我一定错过了什么。您愿意帮我弄清楚这个问题吗？谢谢。

请问您怎么下载的数据集，ADAM和IDRiD数据集，能不能分享一下，万分感谢
